### DISCUSSION TITLE
Вопрос о Wan интерфейсах

### DISCUSSION THREAD

**Discussion by electrifying** [bol-van/zapret#213] [Q&A]
Собственно в чем проблема, если в конфиге zapret указать IFACE_WAN,например, eth3 то заблокированные сайты работают с пк. А вот с телефона нет получаю ошибку err_proxy_connection_failed в конфиге еще есть заккоментированный пункт #IFACE_LAN=eth0 я так понимаю это для работы запрета во всей сети? Как сделать так что бы и на других устройствах заблокированные сайты тоже работали, по команде ifconfig я не вижу ничего связанного с wlan

**Comment by bol-van** [comment]
вчитайтесь в ошибку
там речь идет о прокси
там настроен прокси ?

на какой системе работает запрет?
опенврт или роутер на линухе ?
для опенврт iface_wan игнорируется
на нем  openwrt_wan   и туда пишутся именно опенврт интерфейсы, а не линуховые

**Reply by electrifying** [reply]
На роутере keenetic стоит,вроде там урезанный linux

**Comment by bol-van** [comment]
iface_wan настроить на интерфейс где ип от провайдера
на телефоне разобраться почему лезет через прокси. прокси не нужен
